infra/cifuzz/fuzz_target.py

```diff
@@ -281,18 +280,18 @@ def is_crash_reportable(self, test_case):
     logging.info('The crash is reproducible without the current pull request.')
     return False
 
-  def get_test_case(self, error_string):
+  def get_test_case(self, error_bytes):
```

Comment:
nit: s/error_bytes/error_output ? bytes in var name is slightly weird.

Reply:
I don't like naming things output since it can be confusing, having an argument named "output".
I don't see what is weird here either. Previously the name referred to "string" now since the type is going to be "bytes" I'm changing the name.